lib/rspec-rails.rb

```diff
@@ -8,6 +8,12 @@ module RSpec
   module Rails
     # Railtie to hook into Rails.
     class Railtie < ::Rails::Railtie
+
+      # As of Rails 5.1.0 you can register directories to work with `rake notes`
+      if Gem::Requirement.new(">= 5.1.0").satisfied_by?(Gem::Version.new(::Rails.version))
+        SourceAnnotationExtractor::Annotation.register_directories("spec")
+      end
+      
```

Comment:
The build is breaking due to the whitespace inserted here, our convention is to not allow trailing whitespace in files, would you mind cleaning it up?